unit_tests/common/test_pair.cpp

```diff
@@ -26,8 +26,8 @@ TEST(pair, get_rval_ref) {
     size_t val0 = 1;
     size_t val1 = 2;
 
-    EXPECT_TRUE(
-        std::is_rvalue_reference<decltype(gridtools::get<0>(gridtools::pair<size_t, size_t>{val0, val1}))>::value);
+    static_assert(
```

Comment:
It looks like it is not the part of the contract. I  would just delete this check.

Reply:
done